cmake/Platform/Emscripten-STLab.cmake

```diff
@@ -0,0 +1,138 @@
+#
+# This toolchain file extends `Emscripten.cmake` provided by the Emscripten SDK,
+# and set options required to run STLab test drivers with
+# CTest (using a node runner).
+#
+
+#
+# Find the Emscripten SDK and include its CMake toolchain.
+#
+find_program( EM_CONFIG_EXECUTABLE em-config )
+if ( NOT EM_CONFIG_EXECUTABLE )
+    message( FATAL_ERROR "Could not find emsdk installation. Please install the Emscripten SDK.\nhttps://emscripten.org/docs/getting_started/downloads.html ")
```

Comment:
```suggestion
    message( FATAL_ERROR "Could not find emsdk installation. Please install the Emscripten SDK.\nhttps://emscripten.org/docs/getting_started/downloads.html" )
```

Reply:
Funnily enough - this one was on purpose. I was working around a bug in VSCode: if you click this link it'll take you to `example.com/foo"` and it 404s. But that's the VSCode team's problem.